app/assets/stylesheets/app/components/flip-grid.scss

```diff
@@ -120,7 +120,7 @@
     list-style: none;
     //
     &:hover {
-      background: $black;
+      background: rgba($black,.5);
```

Comment:
`scss-lint` says to do the opposite of both of those. Also, I'm hoping the horrible flip grid will be removed soon anyway, so I'm not going to fix this.